src/main/java/com/suchocki/parkingmeter/restcontroller/ParkingRestController.java

```diff
@@ -0,0 +1,81 @@
+package com.suchocki.parkingmeter.restcontroller;
+
+import java.util.Date;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.InitBinder;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.suchocki.parkingmeter.entity.Driver;
+import com.suchocki.parkingmeter.entity.DriverCharge;
+import com.suchocki.parkingmeter.entity.DriverPayment;
+import com.suchocki.parkingmeter.entity.ParkAction;
+import com.suchocki.parkingmeter.propertyeditor.DatePropertyEditor;
+import com.suchocki.parkingmeter.service.DriverPaymentService;
+import com.suchocki.parkingmeter.service.DriverService;
+
+@RestController
+@RequestMapping("/api")
+public class ParkingRestController {
+
+	@Autowired
+	private DriverService driverService;
+
+	@Autowired
+	private DriverPaymentService driverPaymentService;
+
+	@InitBinder
+	public void initBinder(WebDataBinder binder) {
+		binder.registerCustomEditor(Date.class, new DatePropertyEditor());
+	}
+
+	@PostMapping("/start")
```

Comment:
czy to jest ścieżka zgodna z REST?

Reply:
Nie, zgodna byłaby _/park-actions_. Uznałem jednak, że dla użytkownika będzie mniej czytelna. Gdybym mylił się co do poprawionej wersji (nie jest ona zgodna), proszę o poprawienie mnie.